challenges/01-responsive-web-design/basic-css.json

```diff
@@ -3748,6 +3748,51 @@
         }
       }
     },
+    {
+      "id": "5b7d72c338cd7e35b63f3e14",
+      "title": "Improve Compatibility with Browser Fallbacks",
+      "description": [
+        "When working with CSS you will likely run into browser compatibility issues at some point. This is why it's important to provide browser fallbacks to avoid potential problems.",
+        "When your browser parses the CSS of a webpage, it ignores any properties that it doesn't recognize. For example, if you use a CSS Variable to assign a background color on a site, an Internet Explorer browser will not recognize it because Internet Explorer does not support CSS Variables. In that case, the browser will use whatever value it has for that property. If it can't find any other value set for that property, it will revert to the default value, which is typically not ideal.",
```

Comment:
"When your browser parses the CSS of a webpage, it ignores any properties that it doesn't recognize or support. For example, if you use a CSS variable to assign a background color on a site, Internet Explorer will ignore the background color because it does not support CSS variables. In that case, the browser will use whatever value it has for that property. If it can't find any other value set for that property, it will revert to the default value, which is typically not ideal.",